lib/common/crmcommon_private.h

```diff
@@ -90,6 +90,12 @@ typedef struct xml_doc_private_s {
 G_GNUC_INTERNAL
 void pcmk__xml_set_parent_flags(xmlNode *xml, uint64_t flags);
 
+G_GNUC_INTERNAL
+void pcmk__xml_new_private_data(xmlNode *xml);
+
+G_GNUC_INTERNAL
+void pcmk__xml_free_private_data(xmlNode *xml);
+
```

Comment:
also add an xml section to Pacemaker Development after logging and output, to emphasize the importance of using the wrappers. we can also document the xml/xe/xa/xc scheme